server/src/index.mjs

```diff
@@ -42,9 +42,12 @@ server.on('upgrade', function upgrade(request, socket, head) {
   const { pathname } = parse(request.url);
   let userId = pathname.substring(1);
   if ( ! userId ) {
-    logger.info('Using default user');
-    userId = config.app.defaultUserId;
-  } else if ( ! userManagement.isKnownUser(userId) ) {
+    logger.info('Using ip address as userId');
+    let ipAddresses = socket.remoteAddress
+    userId = ipAddresses.slice(ipAddresses.lastIndexOf(":") + 1);
```

Comment:
This looks like it covers IPv6 addresses only. What about IPv4?

We should be removing all "." in IPv4 addresses and all ":" in IPv6